### ISSUE TITLE
Feature request regarding user list

### ISSUE THREAD

**Issue by waesien** [bigbluebutton/bigbluebutton#13637]
**Is your feature request related to a problem? Please describe.**
This is not a problem, it is a special request.

**Describe the solution you'd like**
In version 2.4, users with poor pc performance will not join the conference late if the same part of loading the server screen is made to the user list after the user joins the conference during screen sharing

After hiding the user list, we did a lesson for 300 people, it was great, we opened the user list, there were those who wanted to enter the lesson later. Those with low pc performance could not attend. They got a exit or wait warning on Chrome.

![image](https://user-images.githubusercontent.com/81984557/140318343-703cd513-ffc5-448d-97d2-c409b5a028f7.png)

With a single camera and sound, it's only possible to hide the user list. I'm aware that over 100 contacts is not recommended, but we teach anyway :)

**Describe alternatives you've considered**
In other words, loading chat messages to the user list or loading the server screen part can be done. "Lazy loading"

![image](https://user-images.githubusercontent.com/81984557/140318757-666be022-a8f6-4efa-a2dd-dc1afc033119.png)

I drew an example

**Comment by waesien** [comment]
![image](https://user-images.githubusercontent.com/81984557/140320296-4d90165c-70fc-46c8-b54a-7109ffa55561.png)

or it can be like this, the user list can be hidden, after the user joined the conference, she can open the user list with the show user list button.

**Comment by antobinary** [comment]
Hi @waesien 

Have you tried passing `userdata-bbb_show_participants_on_login=false` on user join: https://docs.bigbluebutton.org/admin/customize.html#layout-parameters
![Screenshot from 2021-11-04 09-42-20](https://user-images.githubusercontent.com/6312397/140324110-07a07b55-e846-4355-a901-2745ec762e36.png)

This may work well for you as an temporary measure. Please report back if it helped!

**Comment by waesien** [comment]
@antobinary Thank you.

This could be great, I'll test it and get back to you

**Comment by Jossef-767** [comment]
![image](https://user-images.githubusercontent.com/2356928/140420875-3a7f862b-8b3c-4b60-b118-49258504e98d.png)

**Comment by waesien** [comment]
It works, thank you.

I have one last question, is there a parameter in version 2.4 to make the layout type custom instead of automatic?

@antobinary

**Comment by antobinary** [comment]
Nice!

> is there a parameter in version 2.4 to make the layout type custom instead of automatic?

https://github.com/bigbluebutton/bigbluebutton/blob/v2.4.x-release/bigbluebutton-web/grails-app/conf/bigbluebutton.properties#L182-L184

in `/etc/bigbluebutton/bbb-web.properties` override the default by passing
`defaultMeetingLayout=CUSTOM_LAYOUT` and restart BBB
